### PR TITLE
I18n friendly effects

### PR DESCRIPTION
**Pull request by kevingranade**
Consolidated the per-limb effect description messages as per Wuzzy's request: http://smf.cataclysmdda.com/index.php?topic=4403.msg211905#msg211905

### CODE REVIEW

File: data/mods/blazemod/blaze_weapons.json [hunk 1]
```diff
@@ -251,7 +251,7 @@
     "symbol": "(",
     "color": "dark_gray",
     "name": "Dual Nail Turret",
-    "name_plural": "Nail Turrets",
+    "name_plural": "Dual Nail Turrets",
```
Comment (KA101) [inline data/mods/blazemod/blaze_weapons.json]: caps?

**Comment by KA101** [comment]
Misclick there.

**Comment by kevingranade** [comment]
Can lowercase if that's the way we want to go.

**Comment by KA101** [comment]
Well, Rivet lowercased the rest of 'em a while ago, and that is generally the DDA style (names, such as Tesla and Granade, retain uppercase).

**Comment by KA101** [comment]
Merging as-is because string freeze.

**Pull request merged**